pyalgotrading/algobulls/api.py

```diff
@@ -396,16 +396,19 @@ def get_reports(self, strategy_code: str, trading_type: TradingType, report_type
             `GET` v2/user/strategy/statstable       for Stats Table
             `GET` v2/user/strategy/orderhistory     Order History
         """
+        key = self.__get_key(strategy_code=strategy_code, trading_type=trading_type)
         if report_type is TradingReportType.PNL_TABLE:
-            endpoint = 'v2/user/strategy/pltable'
+            endpoint = 'v3/book/pl/data'
+            params = {'pageSize': 0, 'isPythonBuild': "true", 'strategyId': strategy_code}
         elif report_type is TradingReportType.STATS_TABLE:
```

Comment:
we no longer need this `else` block as we are computing stats directly.

Reply:
Done